package.json

```
@@ -7,6 +7,7 @@
   "type": "module",
   "jsdelivr": "dist/chartjs-adapter-luxon.umd.min.js",
   "unpkg": "dist/chartjs-adapter-luxon.umd.min.js",
+  "main": "dist/chartjs-adapter-luxon.umd.min.js",
```

If we're going to add this then it should be the ESM version. Are you on the latest version of the eslint import plugin? It seems like we should file a bug in that repo if so
```suggestion
  "main": "dist/chartjs-adapter-luxon.esm.js",
```

---

@benmccann thanks for commenting!  We are on the latest version of `eslint-plugin-import` as well as eslint.

There's an open issue on `eslint-plugin-import` for this:  https://github.com/import-js/eslint-plugin-import/issues/2132

(You actually commented on the issue thread last year!  😂  https://github.com/import-js/eslint-plugin-import/issues/2132#issuecomment-883769131)

I'm at a conference this week so I'm just skimming the `eslint` comment thread, but it looks like the only workaround is having the main key, even if it's not strictly required.  When I get some downtime I will dive a little deeper in that comment thread.

In the meantime I've updated the reference to `esm` as requested!